schemas/src/registry_schemas/schemas/incorporation.json

```diff
@@ -0,0 +1,76 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/incorporation",
+    "required": [
+        "incorporationHeader",
+        "incorporation"
+    ],
+    "type": "object",
+    "title": "Incorporation Filing",
+    "properties": {
+        "incorporationHeader": {
+            "type": "object",
+            "required": [
+                "name",
+                "date",
+                "certifiedBy"
+            ],
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "title": "The type of the filing.",
+                    "enum": [
+                        "incorporation"
+                    ]
+                },
+                "date": {
+                    "type": "string",
+                    "format": "date",
+                    "title": "The submission date of the final version of the filing."
+                },
+                "certifiedBy": {
+                    "type": "string"
+                }
+            }
+        },
+        "incorporation": {
+            "type": "object",
+            "required": [
+                "nameRequest",
+                "businessContact"
```

Comment:
Is offices an optional element?

Reply:
Good question, it won't be optional i don't believe, but i'm glad you asked because in our current sprint we won't have anything with regards to adding the office addresses component and data into the POST, so i left it as optional until we can implement that piece. Would it make sense to leave it out until we have it, or just enforce it now?

Reply:
I am ok with adding it later :).